webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -40,13 +40,19 @@ export const CategorySelector = (props) => {
   const [showCategoriesPanel, setShowCategoriesPanel] = useState(false)
   const [categoryToEdit, setCategoryToEdit] = useState(null)
 
-  const emptyItem = { uuid: null, label: i18n.t('common.notSpecified') }
+  const emptyItem = useMemo(() => ({ uuid: null, label: i18n.t('common.notSpecified') }), [i18n])
```

Comment:
how often does the i18n change when this component is rendered?

Reply:
never, it was just to avoid eslint to complain about a missing dependency!

Reply:
It shouldn't cry before,

Reply:
I'll remove it, it's unnecessary